resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -278,6 +278,13 @@
 );
 ?>
 
+<?php if (!$isFullyFeaturedGame): ?>
+    <?php
+        $pageType = 'retroachievements:hub';
+        $pageImage = media_asset($gameData['ImageIcon']);;
+    ?>
+<?php endif ?>
```

Comment:
Is there a reason why new code would use `<?php if (): ?>` instead of `@if ()`?

Reply:
Not really, I just made it the same style as the other if statement below it, I've updated both of them.